### ISSUE TITLE
Update darkstat and ejabberd

### ISSUE THREAD

**Issue by netzwanze** [SynoCommunity/spksrc#4214]
NAS Model: DS620slim
NAS Architecture:  (x64)
DSM version: DSM 6.2.3-25426 Update 2

### Expected behavior
I migrated  my NAS from DS416play to DS620slim -> some packages are no longer displayed
DarkStat, ejabberd,ncdu

### Actual behavior
I migrated my NAS the packages was brocken on the new NAS. I have deinstalled the packages.
Now, i can not reinstall, because the packages are longer displayed 

#### Package log
_Check Package Center or `/usr/local/{package}/var/`_
```
not exist
```
#### Other logs as sample for all packages it was installed and now not exist in teh package center
```
DS620:~$ cat /var/log/synopkg.log| grep -i jabb
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin preinst
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End preinst ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/rm -rf /volume1/@appstore/ejabberd
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/rm -rf /volume1/@appstore/ejabberd ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/ejabberd
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/ejabberd ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/rm -rf /var/packages/ejabberd
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/rm -rf /var/packages/ejabberd ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/mkdir -p /var/packages/ejabberd
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/mkdir -p /var/packages/ejabberd ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/ejabberd/INFO
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/ejabberd/INFO ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/rm -rf /var/packages/ejabberd/scripts
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/rm -rf /var/packages/ejabberd/scripts ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/ejabberd/scripts
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/ejabberd/scripts ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/rm -rf /var/packages/ejabberd/WIZARD_UIFILES
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/rm -rf /var/packages/ejabberd/WIZARD_UIFILES ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/ejabberd/WIZARD_UIFILES
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/ejabberd/WIZARD_UIFILES ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/rm -rf /var/packages/ejabberd/conf
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/rm -rf /var/packages/ejabberd/conf ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/ejabberd/conf
2020/09/11 21:39:41	install ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/ejabberd/conf ret=[0]
2020/09/11 21:39:41	install ejabberd 2.1.13-1 Begin postinst
2020/09/11 21:41:19	install ejabberd 2.1.13-1 End postinst ret=[0]
2020/09/11 21:41:19	install ejabberd 2.1.13-1 Begin /bin/rm -rf /volume1/@tmp/pkginstall
2020/09/11 21:41:19	install ejabberd 2.1.13-1 End /bin/rm -rf /volume1/@tmp/pkginstall ret=[0]
2020/09/11 21:41:19	install ejabberd 2.1.13-1 successfully
2020/09/11 21:41:21	install ejabberd: begin to start version 2.1.13-1
2020/09/11 21:41:21	install ejabberd 2.1.13-1 Begin pre-load apparmor
2020/09/11 21:41:21	install ejabberd 2.1.13-1 End pre-load apparmor ret=[0]
2020/09/11 21:41:21	install ejabberd 2.1.13-1 Begin start-stop-status start
2020/09/11 21:42:52	install ejabberd 2.1.13-1 End start-stop-status start ret=[1]
2020/09/11 21:42:53	(system) trigger ejabberd 2.1.13-1 Begin start-stop-status stop
2020/09/11 21:42:53	(system) trigger ejabberd 2.1.13-1 End start-stop-status stop ret=[0]
2020/09/11 21:42:53	(system) trigger ejabberd 2.1.13-1 Begin unload apparmor
2020/09/11 21:42:53	(system) trigger ejabberd 2.1.13-1 End unload apparmor ret=[0]
2020/09/11 21:42:53	stop ejabberd: begin to stop version 2.1.13-1
2020/09/11 21:42:53	stop ejabberd: stop version 2.1.13-1 successfully, result 0
2020/09/11 21:42:53	install ejabberd: start version 2.1.13-1 failed, result 272
2020/09/11 22:07:34	upgrade ejabberd 2.1.13-1 Begin preupgrade
2020/09/11 22:07:34	upgrade ejabberd 2.1.13-1 End preupgrade ret=[0]
2020/09/11 22:07:34	upgrade ejabberd 2.1.13-1 Begin preuninst
2020/09/11 22:07:34	upgrade ejabberd 2.1.13-1 End preuninst ret=[0]
2020/09/11 22:07:34	upgrade ejabberd 2.1.13-1 Begin /bin/rm -rf /volume1/@appstore/ejabberd
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 End /bin/rm -rf /volume1/@appstore/ejabberd ret=[0]
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 Begin postuninst
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 End postuninst ret=[0]
2020/09/11 22:07:36	upgrade ejabberd: Uninstall 2.1.13-1 successfully
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 Begin preinst
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 End preinst ret=[0]
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 Begin /bin/rm -rf /volume1/@appstore/ejabberd
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 End /bin/rm -rf /volume1/@appstore/ejabberd ret=[0]
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/ejabberd
2020/09/11 22:07:36	upgrade ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/package /volume1/@appstore/ejabberd ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/mkdir -p /var/packages/ejabberd
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/mkdir -p /var/packages/ejabberd ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/ejabberd/INFO
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/INFO /var/packages/ejabberd/INFO ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/rm -rf /var/packages/ejabberd/scripts
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/rm -rf /var/packages/ejabberd/scripts ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/ejabberd/scripts
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/scripts /var/packages/ejabberd/scripts ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/rm -rf /var/packages/ejabberd/WIZARD_UIFILES
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/rm -rf /var/packages/ejabberd/WIZARD_UIFILES ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/ejabberd/WIZARD_UIFILES
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/WIZARD_UIFILES /var/packages/ejabberd/WIZARD_UIFILES ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/rm -rf /var/packages/ejabberd/conf
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/rm -rf /var/packages/ejabberd/conf ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/ejabberd/conf
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 End /bin/mv -f /volume1/@tmp/pkginstall/conf /var/packages/ejabberd/conf ret=[0]
2020/09/11 22:07:37	upgrade ejabberd 2.1.13-1 Begin postinst
2020/09/11 22:07:41	upgrade ejabberd 2.1.13-1 End postinst ret=[0]
2020/09/11 22:07:41	upgrade ejabberd 2.1.13-1 Begin postupgrade
2020/09/11 22:07:41	upgrade ejabberd 2.1.13-1 End postupgrade ret=[0]
2020/09/11 22:07:41	upgrade ejabberd from 2.1.13-1 to 2.1.13-1 successfully
2020/09/11 22:07:41	upgrade ejabberd 2.1.13-1 Begin /bin/rm -rf /volume1/@tmp/pkginstall
2020/09/11 22:07:41	upgrade ejabberd 2.1.13-1 End /bin/rm -rf /volume1/@tmp/pkginstall ret=[0]
2020/09/11 22:07:44	install ejabberd: begin to start version 2.1.13-1
2020/09/11 22:07:44	install ejabberd 2.1.13-1 Begin pre-load apparmor
2020/09/11 22:07:44	install ejabberd 2.1.13-1 End pre-load apparmor ret=[0]
2020/09/11 22:07:44	install ejabberd 2.1.13-1 Begin start-stop-status start
2020/09/11 22:09:16	install ejabberd 2.1.13-1 End start-stop-status start ret=[1]
2020/09/11 22:09:16	(system) trigger ejabberd 2.1.13-1 Begin start-stop-status stop
2020/09/11 22:09:16	(system) trigger ejabberd 2.1.13-1 End start-stop-status stop ret=[0]
2020/09/11 22:09:16	(system) trigger ejabberd 2.1.13-1 Begin unload apparmor
2020/09/11 22:09:16	(system) trigger ejabberd 2.1.13-1 End unload apparmor ret=[0]
2020/09/11 22:09:17	stop ejabberd: begin to stop version 2.1.13-1
2020/09/11 22:09:17	stop ejabberd: stop version 2.1.13-1 successfully, result 0
2020/09/11 22:09:17	install ejabberd: start version 2.1.13-1 failed, result 272
2020/10/08 18:25:34	uninstall ejabberd 2.1.13-1 Begin preuninst
2020/10/08 18:25:34	uninstall ejabberd 2.1.13-1 End preuninst ret=[0]
2020/10/08 18:25:34	uninstall ejabberd 2.1.13-1 Begin /bin/rm -rf /volume1/@appstore/ejabberd
2020/10/08 18:25:36	uninstall ejabberd 2.1.13-1 End /bin/rm -rf /volume1/@appstore/ejabberd ret=[0]
2020/10/08 18:25:36	uninstall ejabberd 2.1.13-1 Begin postuninst
2020/10/08 18:25:36	uninstall ejabberd 2.1.13-1 End postuninst ret=[0]
2020/10/08 18:25:36	uninstall ejabberd: Uninstall 2.1.13-1 successfully
```

**Comment by publicarray** [comment]
By changing your hardware the CPU architecture for packages has also changed from `braswell` (DS416play) to `apollolake` (DS620slim) You can try and install the x64 versions manually from the website:

* [DarkStat](https://synocommunity.com/package/darkstat) does not appear to be there for your new architecture needs to be re-published. This looks like an older package anyway @ymartin59 
* Same with [ejabberd](https://synocommunity.com/package/ejabberd) & hasn't been updated in a long time
* ncdu is now part of the [synocli-disk](https://synocommunity.com/package/synocli-disk) package

**Comment by hgy59** [comment]
Finally darkstat is updated and released for all supported arch see #5224 (it will never be compatible with DSM 7).
